runtime/service.go

```diff
@@ -482,10 +482,10 @@ func (s *service) CreateVM(requestCtx context.Context, request *proto.CreateVMRe
 	if err != nil {
 		s.shimCancel()
 		s.logger.WithError(err).Error("failed to create VM")
-		if errors.Cause(err) == context.DeadlineExceeded {
+		if strings.Contains(err.Error(), context.DeadlineExceeded.Error()) {
```

Comment:
@kzys everything else was pretty simple changes, but I wasn't sure the idiomatic change here. No longer wrapping errors so I assumed a substring check.

Reply:
`%w` in the format string wraps the error. The way to check is `errors.Is(err, context.DeadlineExceeded)`

Reply:
Oh, +1. I had only checked errors.go should have checked wrap.go as well. :)